### PR TITLE
[ISSUE #17] Corrected the angle functions to fix the incorrected results strans function according to the documentation.

### PR DESCRIPTION
**Pull request by stepan-neretin7**
The problem was that our corners are float8, and for some reason pointers were returned.

### CODE REVIEW

**Review by esabol — APPROVED** [review]

**Review by vitcpp — APPROVED** [review]

**Comment by vitcpp** [comment]
I propose to change version to 1.2.3 (increment patch number) before merge.

@stepan-neretin7 Could you please change the version number to 1.2.3 in the following places:
- Makefile: change PGSPHERE_VERSION to 1.2.3
- pg_sphere.control: change default_version to 1.2.3

File: sql/euler.sql [hunk 1]
```diff
@@ -1,9 +1,55 @@
...
+
+SELECT phi( - strans ( '20d, 50d, 80d, XYZ' ) );
+
+SELECT strans( '90d, 60d, 30d' );
+
+SELECT theta( strans( '90d, 60d, 30d' ) );
+
+SELECT psi( strans( '90d, 60d, 30d' ) );
+
+SELECT phi( strans( '90d, 60d, 30d' ) );
```
Comment (esabol) [inline sql/euler.sql:55]: You're missing a linefeed at the end of this file. This is a POSIX requirement.

**Review by esabol — CHANGES_REQUESTED** [review]

File: sql/euler.sql (hunk 1 above)
Comment (stepan-neretin7) [reply sql/euler.sql]: fixed it

**Comment by vitcpp** [comment]
@stepan-neretin7 Thank you for the fix! It works on my side. I think the patch is ready for merge.

The decision to increment version number was made because the fix is important. I propose to define version 1.2.3 as stable and assign the tag '1.2.3'. Stub upgrade script was added to make upgrade process to complete without errors.

**Review by esabol — APPROVED** [review]

**Comment by esabol** [comment]
Looks good to me! Thank you!

**Pull request merged**